uni/lib/view/common_widgets/pages_layouts/general/widgets/top_right_button.dart

```diff
@@ -0,0 +1,60 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:uni/model/providers/startup/profile_provider.dart';
+import 'package:uni/model/providers/startup/session_provider.dart';
+import 'package:uni/view/common_widgets/pages_layouts/general/general.dart';
+import 'package:uni/view/profile/profile.dart';
+
+class TopRightButton extends StatelessWidget {
+  const TopRightButton({required this.getProfileDecorationImage, super.key});
```

Comment:
Should this class be named TopRightButton? I think you should name it something more semantic eg.: ProfileButton to make it more maintainable long term, what do you think?

Reply:
Yes, I just reused the old name. I will change it.

Reply:
Done :)